src/components/SlateEditor/utils/mergeLastUndos.ts

```diff
@@ -2,8 +2,14 @@ import { Editor } from 'slate';
 
 const mergeLastUndos = (editor: Editor) => {
   const arr = editor.history.undos;
-  const tail = arr.pop() || [];
-  arr[arr.length - 1] = arr[arr.length - 1].concat(tail);
+  const newest = arr.pop();
+  const older = arr[arr.length - 1];
+
+  const newObject = {
+    operations: older?.operations.concat(newest?.operations || []) || newest?.operations || [],
+    selectionBefore: older.selectionBefore || newest?.selectionBefore || null,
+  };
+  arr[arr.length - 1] = newObject;
```

Comment:
Dette er bare en merge av old og new, ved å legge til de nyeste undo'ene i det siste elementet?

Reply:
Noen av embeden er implementert litt upraktisk (burde vi kanskje endre på sikt).

Feks vil innsetting av forklaring bestå av to operasjoner:
1. Sette inn en forklaring uten id. (modal åpnes automatisk når id ikke finnes)
2. Bruker velger en forklaring, det gjøres en Transform for å legge id på forklaringen

For brukeren oppleves dette kun som en operasjon, og en undo vil i utgangspunktet ta deg tilbake til steg 1 og dermed åpne modalen igjen. Derfor merges eventene slik at både stefg 1 og 2 blir omgjort ved undo.